plot_features.py

```diff
@@ -1,122 +1,98 @@
-        map.drawparallels(np.arange(-80,81,20),labels=[1,1,0,0])
-        map.drawmeridians(np.arange(0,360,60),labels=[0,0,0,1])
-    elif maptype == 'vandg':
-        map = Basemap(projection='vandg',lon_0=0,resolution='l')
-        map.drawparallels(np.arange(-80,81,20))
-        map.drawmeridians(np.arange(0,360,60))
-    elif maptype == 'mill':
-        map = Basemap(llcrnrlon=0,llcrnrlat=-90,urcrnrlon=360,urcrnrlat=90,projection='mill', resolution='l')
-        map.drawparallels(np.arange(-80,81,20),labels=[1,1,0,0])
-        map.drawmeridians(np.arange(0,360,60),labels=[0,0,0,1])
-    elif maptype == 'mill2':
-        map = Basemap(llcrnrlon=-180,llcrnrlat=-90,urcrnrlon=180,urcrnrlat=90,projection='mill', resolution='l')
-        map.drawparallels(np.arange(-80,81,20),labels=[1,1,0,0])
-        map.drawmeridians(np.arange(0,360,60),labels=[0,0,0,1])
-    elif maptype == 'robin':
-        map = Basemap(projection='robin',lon_0=0,lat_0=0, resolution='l')
-        map.drawparallels(np.arange(-80,81,20),labels=[1,1,0,0])
-        map.drawmeridians(np.arange(0,360,60),labels=[0,0,0,1])
-    elif maptype == 'robin2':
-        map = Basemap(projection='robin',lon_0=180,lat_0=0, resolution='l')
-        map.drawparallels(np.arange(-80,81,20),labels=[1,1,0,0])
-        map.drawmeridians(np.arange(0,360,60),labels=[0,0,0,1])
-    elif maptype == 'hammer':
-        map = Basemap(projection='hammer',lon_0=0, resolution='l')
-        map.drawmeridians(np.arange(0,360,30))
-        map.drawparallels(np.arange(-90,90,30))
-    elif maptype == 'northpole':
-        map = Basemap(projection='ortho', lat_0=90, lon_0=-180, resolution='l')
-        map.drawmeridians(np.arange(0,360,30))
-        map.drawparallels(np.arange(-90,90,30))
-    elif maptype == 'southpole':
-        map = Basemap(projection='ortho', lat_0=-90, lon_0=0, resolution='l')
-        map.drawmeridians(np.arange(0,360,30))
-        map.drawparallels(np.arange(-90,90,30))
-    elif maptype == 'atlantic':
-        map = Basemap(projection='ortho', lat_0=0, lon_0=0, resolution='l')
-        map.drawmeridians(np.arange(0,360,30))
-        map.drawparallels(np.arange(-90,90,30))
-    elif maptype == 'pacific':
-        map = Basemap(projection='ortho', lat_0=0, lon_0=-180, resolution='l')
-        map.drawmeridians(np.arange(0,360,30))
-        map.drawparallels(np.arange(-90,90,30))
-    elif maptype == 'americas':
-        map = Basemap(projection='ortho', lat_0=0, lon_0=-90, resolution='l')
-        map.drawmeridians(np.arange(0,360,30))
-        map.drawparallels(np.arange(-90,90,30))
-    elif maptype == 'asia':
-        map = Basemap(projection='ortho', lat_0=0, lon_0=90, resolution='l')
-        map.drawmeridians(np.arange(0,360,30))
-        map.drawparallels(np.arange(-90,90,30))
+import os.path as ospath
+
+import cartopy.crs
+import cartopy.feature
+
+
+def plot_base(mapType): #{{{
+
+    if mapType == 'cyl':
```

Comment:
@pelson, I think you're right.  My initial thought was that we only want to build the projections we're actually going to use, but they don't seem to be expensive to build.  I'll make this change.